migration/src/jira_util.py

```diff
@@ -83,6 +83,15 @@ def extract_assignee(o: dict) -> tuple[str, str]:
     return (name, disp_name)
 
 
+def extract_parent(o: dict) -> tuple[str, str]:
+    parent = o["fields"].get("parent")
+    if parent:
+        key = parent["key"]
+        if key:
+            return key, f'https://issues.apache.org/jira/browse/{key}'
```

Comment:
I would just extract the key and make the url on-the-fly at `jir2github_import.py` (L120).

Reply:
Ahh good idea, will do.